lib/ts/types.ts

```diff
@@ -27,6 +27,7 @@ export type Event =
 export type EventHandler = (event: Event) => void;
 
 export type InputType = {
+    enableDebugLogs?: boolean;
```

Comment:
add tests where we set this to true, and check that logging is happening.. and that if we set it to false or don't set it, logging is not happening

Reply:
Added tests for all of the above mentioned cases